docs/api-docs/storefront/graphql/graphql-faceted-textual-search.md

```diff
@@ -0,0 +1,612 @@
+                       productCount
+                       subCategories {
+                         edges {
+                           node {
+                             entityId
+                             name
+                           }
+                         }
+                       }
+                     }
+                   }
+                 }
+               }
+               ... on OtherSearchFilter {
+                 name
+                 displayProductCount
+                 isCollapsedByDefault
+                 freeShipping {
+                   isSelected
+                   productCount
+                 }
+                 isInStock {
+                   isSelected
+                   productCount
+                 }
+                 isFeatured {
+                   isSelected
+                   productCount
+                 }
+               }
+             }
+           }
+         }
+      }
+    }
+  }
+}
+```
+<!--
+type: tab
+title: Response
+-->
+
+```json title="Example" lineNumbers
+{
+  "data": {
+    "site": {
+      "search": {
+        "searchProducts": {
+          "filters": {
+            "edges": [
+              {
+                "node": {
+                  "__typename": "CategorySearchFilter",
+                  "name": "Category",
+                  "isCollapsedByDefault": false,
+                  "displayProductCount": true,
+                  "categories": {
+                    "edges": [
+                      {
+                        "node": {
+                          "entityId": 19,
+                          "isSelected": false,
+                          "productCount": 1,
+                          "subCategories": {
+                            "edges": []
+                          }
+                        }
+                      },
+                      {
+                        "node": {
+                          "entityId": 21,
+                          "isSelected": false,
+                          "productCount": 1,
+                          "subCategories": {
+                            "edges": []
+                          }
+                        }
+                      }
+                    ]
+                  }
+                }
+              },
+              {
+                "node": {
+                  "__typename": "OtherSearchFilter",
+                  "name": "Other",
+                  "isCollapsedByDefault": true,
+                  "displayProductCount": true,
+                  "freeShipping": null,
+                  "isInStock": {
+                    "isSelected": false,
+                    "productCount": 2
+                  },
+                  "isFeatured": null
+                }
+              }
+            ]
+          }
+        }
+      }
+    }
+  }
+}
+```
+<!-- type: tab-end -->
+
+For a complete list of facets that can be returned, see the [GraphQL Storefront Playground](/graphql-playground).
```

Comment:
```suggestion
For a complete list of facets that can be returned, see the [GraphQL Storefront Playground](https://developer.bigcommerce.com/graphql-playground).
```
the playground isn't part of the Stoplight docs app, so it needs the whole URL. (this is also makes it open in a new window, which is great IMO for its use case.)